docs/concepts.md

```diff
@@ -99,6 +99,24 @@ anticipated to be a rating provided by the Ratings Provider separately, such as
 the effective seasonal rating, a verbal override, or even a previously
 forecasted rating.
 
+### Seasonal Ratings
```

Comment:
We recently added https://trolie.energy/concepts#seasonal-overrides
I'd suggested moving this to just above that after rebasing.

Reply:
Done.